src/scss/tools/mixins/_grid.scss

```diff
@@ -71,6 +91,10 @@
     margin-left: percentage($gutter / 2);
   }
 
+  .hide-#{$key} {
```

Comment:
@GeorgeJegon we need to scope `hide` selector (`hide-small`, `hide-medium`, etc.) and this selectors needs to be inside `min-width: breakpoint1` and `max-width: breakpoint2`